Dockerfile

```diff
@@ -1,8 +1,29 @@
-FROM node:18-alpine
+FROM --platform=${BUILDPLATFORM} node:20-slim as builder
```

Comment:
copied this from [helia-http-gateway](https://github.com/ipfs/helia-http-gateway/blob/87deb922f6137b0a288954e6de149107e0f7a648/Dockerfile) and removed unnecessary items.